shmem.c

```diff
@@ -175,9 +179,11 @@ void lock_shm() {
 		logg("Failed to obtain SHM lock: %s", strerror(result));
 }
 
-void unlock_shm() {
+void _unlock_shm(const char* function, const int line, const char * file) {
```

Comment:
Provided by a macro in `shmem.h`. Note the `_` at the beginning of the name to symbolize this.